types.ts

```diff
@@ -55,10 +65,13 @@ export interface Frame extends Computation<Result<void>> {
   destroy(): Computation<Result<void>>;
 }
 
-export type BlockResult<T> = Result<T> | {
-  type: "aborted";
-  result: Result<void>;
-};
+export type BlockResult<T> =
+  | Result<T>
+  | {
```

Comment:
This is a little awkward now but it's kind of nice that we still have the `type` property so we can extend the `Result` if needed with additional properties.  Thoughts?